src/System.Windows.Forms.Primitives/tests/UnitTests/Windows/Win32/GlobalInterfaceTableTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using Windows.Win32.System.Com;
+
+namespace System.Windows.Forms.Primitives.Tests.Windows.Win32
+{
+    public unsafe class GlobalInterfaceTableTests
+    {
+        [Fact]
+        public void GlobalInterfaceTable_CheckRefCounts()
+        {
+            using var stream = ComHelpers.GetComScope<IStream>(new MyStream(), out bool success);
+            Assert.True(success);
+            uint count = stream.Value->AddRef();
+            Assert.Equal(2u, count);
+            count = stream.Value->Release();
+            Assert.Equal(1u, count);
+            uint cookie = GlobalInterfaceTable.RegisterInterface(stream.Value);
+            count = stream.Value->AddRef();
+            // +1 for the AddRef, and +1 for the git registration
+            Assert.Equal(3u, count);
+            count = stream.Value->Release();
+            Assert.Equal(2u, count);
+            GlobalInterfaceTable.RevokeInterface(cookie);
+            count = stream.Value->AddRef();
+            Assert.Equal(2u, count);
+            stream.Value->Release();
+        }
```

Comment:
Please consider adding blank lines to help with readability
```suggestion
        public void GlobalInterfaceTable_CheckRefCounts()
        {
            using var stream = ComHelpers.GetComScope<IStream>(new MyStream(), out bool success);
            Assert.True(success);

            uint count = stream.Value->AddRef();
            Assert.Equal(2u, count);

            count = stream.Value->Release();
            Assert.Equal(1u, count);

            uint cookie = GlobalInterfaceTable.RegisterInterface(stream.Value);
            count = stream.Value->AddRef();
            // +1 for the AddRef, and +1 for the git registration
            Assert.Equal(3u, count);

            count = stream.Value->Release();
            Assert.Equal(2u, count);

            GlobalInterfaceTable.RevokeInterface(cookie);
            count = stream.Value->AddRef();
            Assert.Equal(2u, count);
            stream.Value->Release();
        }
```